src/app/frontend/chrome/chrome.scss

```diff
@@ -30,10 +30,16 @@
   min-height: 7 * $baseline-grid;
 }
 
+.kd-content-div {
+  height: 100%;
+}
+
 .kd-spinner {
   align-content: center;
   align-items: center;
+  display: flex;
```

Comment:
Yeah, it wasn't so obvious, but I remembered it from one of my previous changes :)